packages/terra-embedded-content-consumer/src/EmbeddedContentConsumer.module.scss

```diff
@@ -0,0 +1,15 @@
+// Themes
+@import './clinical-lowlight-theme/EmbeddedContentConsumer.module';
+@import './orion-fusion-theme/EmbeddedContentConsumer.module';
+
+:local {
+  /* stylelint-disable terra/custom-property-name */
+  $blur-outline: var(--terra-embedded-content-consumer-blur-outline, none);
+  $focus-outline: var(--terra-embedded-content-consumer-focus-outline, 2px dashed #000);
+
+  :export {
+    /* stylelint-disable property-no-unknown */
+    blurOutline: $blur-outline;
+    focusOutline: $focus-outline;
+  }
```

Comment:
You should be able to set the styles using classes.  I don't think that this is the correct implementation.

Reply:
These are not for the classes to be applied to an html element like we normally do.

Reply:
You would add the class to the iFrame element.